email/ally-fulfilment-inline.html

```diff
@@ -0,0 +1,232 @@
+                                                        <table border="0" cellpadding="0" cellspacing="0" class="main-content" style="border-collapse:collapse;mso-table-lspace:0pt;mso-table-rspace:0pt;width:100%;text-align:left;">
+                                                            <tbody>
+                                                                <tr>
+                                                                    <td class="img" style="border-collapse:collapse;font-family:'Source Sans Pro', sans-serif;font-size:16px;vertical-align:top;width:49%;vertical-align:middle;padding-bottom:19px;padding-right:8px;">
+                                                                        <a href="#" title="Book a reporting run-through" target="_blank" style="color:#191F25;text-decoration:underline;"><img src="_assets/img/img-reporting.png" alt="Book a session to run through reporting" width="289" style="-ms-interpolation-mode:bicubic;max-width:100%;outline:none;text-decoration:none;-ms-interpolation-mode:bicubic;border:none;"></a>
+                                                                    </td>
+                                                                    <td class="message" style="border-collapse:collapse;font-family:'Source Sans Pro', sans-serif;font-size:16px;vertical-align:top;">
+                                                                        <p style="font-family:'Source Sans Pro', sans-serif;font-size:16px;font-weight:normal;margin:0;color:#191F25;margin:1em 0;font-size:18px;padding-right:32px;vertical-align:middle;">Schedule a time to run through the institutional report together.</p>
+                                                                        <table border="0" cellpadding="0" cellspacing="0" class="btn btn-primary" style="border-collapse:collapse;mso-table-lspace:0pt;mso-table-rspace:0pt;box-sizing:border-box;width:100%;text-align:center;padding:16px;text-align:left;">
+                                                                            <tbody>
+                                                                                <tr>
+                                                                                    <td style="border-collapse:collapse;font-family:'Source Sans Pro', sans-serif;font-size:16px;vertical-align:top;">
+                                                                                        <a href="#" title="Book a reporting run-through" target="_blank" style="color:#191F25;text-decoration:underline;background-color:#FFFFFF;border:solid 1px #007900;box-sizing:border-box;color:#007900;cursor:pointer;display:inline-block;font-size:15px;font-weight:bold;line-height:1.6;margin:0;padding:8px 16px;text-decoration:none;text-transform:capitalize;background-color:#44474F;border-color:#44474F;color:#FFFFFF;">Book now</a>
```

Comment:
Also, if it's actually performing an action to "Book now", perhaps it should be a `button` instead. If it's just taking them to a website to book though then a link is fine I think.